client/src/main.rs

```diff
@@ -1035,6 +1037,52 @@ fn override_endpoint(
     Ok(())
 }
 
+fn prompt_override_endpoint(
+    info: &Info,
+    args: &OverrideEndpointOpts,
+    listen_port: u16,
+) -> Result<Option<Endpoint>, Error> {
+    let endpoint = match &args.endpoint {
+        Some(endpoint) => endpoint.clone(),
```

Comment:
I guess we can/should bail out if an unspecified addr is set _on the command line_ and the server doesn't support that.

Reply:
@mbernat I meant when someone types: `innernet override-endpoint --endpoint 0.0.0.0:1234` -- I think we don't catch this case currently and let it fall through to the (old) server.

Reply:
```
marek@Mareks-Laptop innernet % cargo build -q && sudo ./target/debug/innernet -v override-endpoint --endpoint 0.0.0.0:1 tabletop
[D] innernet server endpoint capabilities not found, assuming default capabilities

[E] Attempted to use an unspecified IP (all zeros) but the innernet server does
    not have the capability to resolve it, likely because its version is older.
```